src/cpu.rs

```diff
@@ -35,8 +35,10 @@ pub(crate) fn features() -> Features {
         )
     ))]
     {
-        static INIT: spin::Once<()> = spin::Once::new();
-        let () = INIT.call_once(|| {
+        use once_cell::sync::OnceCell;
+
+        static INIT: OnceCell<bool> = OnceCell::new();
+        let _ = INIT.get_or_init(|| {
```

Comment:
This will not work for a `no_std` environment, though, right?

I think the right thing to do is use `std::sync::Once` or `OnceCell` for environments that have `libstd` but something else for `no_std` environments.